src/vue_isotope.js

```diff
@@ -115,18 +117,18 @@
                 this.iso.updateSortData();
                 this.iso._requestUpdate();
               });
-            });  
+            });
           }).flatten().value();
         },
 
         sort (name) {
-          this.arrange({sortBy  :name})
+          this.arrange({sortBy: name, sortAscending: this.options.sortAscending})
```

Comment:
Leaving sortAscending undefined still allows for sorting and changing which parameter to sort on (ex. rating vs length) but it was impossible to change the sort order from ascending to descending from within the vue componant data (from highest rating first to lowest first for example). Setting `  sortAscending: this.sortAscending,` to true/false has no effect.